components/test_raftstore/src/cluster.rs

```diff
@@ -733,7 +733,8 @@ impl<T: Simulator> Cluster<T> {
     ) -> RaftCmdResponse {
         let timer = Instant::now();
         let mut tried_times = 0;
-        while tried_times < 10 || timer.elapsed() < timeout {
+        // At least retry once.
+        while tried_times < 2 || timer.elapsed() < timeout {
```

Comment:
Looks like it is `at most retry once` (if the timer does not fire first), also why we need this change?

Reply:
Without the patch, it can retry at least 10 times. Some operation inside the loop can take more than 5 seconds, so the method can wait for 50 seconds, which is too long in the test of hibernate regions and exceeds hibernate timeout eventually.

Reply:
Perhaps we should change `||` to `&&`?

Reply:
Using `||` is OK. It's possible that the first try stuck for a while in TiKV and fail. Make it try another time in this case should make cases stable.

Reply:
I mean keeping the try times at 10 and use `||`.